packages/near-membrane-base/src/membrane.ts

```diff
@@ -257,43 +267,26 @@ export function createMembraneMarshall() {
             shadowTarget: ShadowTarget,
             foreignTargetPointer: Pointer
         ) {
-            let keys: ReturnType<typeof Reflect.ownKeys> = [];
-            foreignCallableOwnKeys(foreignTargetPointer, (...args) => {
```

Comment:
With this change we avoid a lot of boundary calls and perform just 1.

Reply:
The old way did a `foreignCallableOwnKeys` call then a `foreignCallableGetOwnPropertyDescriptor` per key. The new way just does 1 `foreignCallableGetOwnPropertyDescriptors` call.